app/.env.local.example

```diff
@@ -0,0 +1,3 @@
+MONGODB_DATA_API_KEY="<Data_API_Key>"
+MONGODB_DATA_API_URL="<Data_API_Endpoint_URL>"
```

Comment:
the file (or rather `.env.local`) serves for the environment variables (secret credentials) in NextJS, you'll have to add the credentials locally, link sent privately